packages/components/src/components/event/eventInfo/EventsHierarchy.tsx

```diff
@@ -90,13 +96,19 @@ const SubEvents: React.FC<{ event: EventFields }> = ({ event }) => {
     <div className={styles.eventList}>
       <InfoWithIcon icon={titleIcon} title={title}>
         {isLowestLevelEvent || isMiddleLevelEvent ? (
-          <EventList id={subEventsListTestId} events={shownEvents} showDate />
+          <EventList
+            id={subEventsListTestId}
+            events={shownEvents}
+            showDate
+            getEventListLinkUrl={getEventListLinkUrl}
```

Comment:
The getEventListLinkUrl would have been better to be offered with a React context instead